eciML1515/scripts/fitGAM.m

```diff
@@ -0,0 +1,93 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% GAM = fitGAM(model)
+% Returns a fitted GAM for the yeast model.
+% 
+% Ivan Domenzain. Last update: 2019-04-24
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+function GAM = fitGAM(model)
```

Comment:
why the need of this function? It is not used in this branch's version of `scaleBioMass.m` so could be skipped entirely

Reply:
Thanks for noticing this, the function has now been removed by 
b7e0131